cupyx/scipy/interpolate/_bspline2.py

```diff
@@ -0,0 +1,509 @@
+import operator
+
+from numpy.core.multiarray import normalize_axis_index
+
+import cupy
+from cupyx.scipy import sparse
+from cupyx.scipy.sparse.linalg import spsolve
+
+from ._bspline import _get_module_func, INTERVAL_MODULE, D_BOOR_MODULE, BSpline
```

Comment:
We don't use relative module paths, so would you rewrite this to an absolute one?

Reply:
Good to know, thanks! Done in https://github.com/cupy/cupy/pull/7195/commits/b08a5989960ac77e69178fc0c66cb0f81f231cee